bokehjs/src/lib/models/axes/axis.ts

```diff
@@ -287,20 +326,25 @@ export class AxisView extends GuideRendererView {
     const c = Math.cos(angle)
```

Comment:
@bryevdv, what's the reason to have a different scaling factor depending on the type of the orientation? I'm speaking about `hscale`, but github doesn't allow me to place this comment in the right place.

Reply:
My only recollection is that it is a fudge factor that more or less lets things work when the orientation was angled at all. The real problem could be in `get_text_height` which is just an approximating function we created, IIRC, since browsers inexplicably don't report text height reliably. I.e. maybe `get_text_height` underestimates consistently, and things look OK for orthogonal orientations, but bad for angled ones, and if it computed a better value `hscale` would be unneeded. Could try setting `hscale` to 1.5 unconditionally and see if things look ok in both orthogonal/angled cases (that would simulate correcting for text height underestimate)